src/main/webapp/index.html

```diff
@@ -5,40 +5,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  
     <title>GPostCard</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/env.css">
```

Comment:
any reason for having a different stylesheet for same index file?
e.g. container class style is declared in both file, style is assigned in cascade

Reply:
Yes, just easier to have a big css design for an envelope. It is quite complex

Reply:
and container is just relevant to both

Reply:
it's ok to use multiple stylesheets in a single template file, if each file is organized for a specific purpose.
e.g. env stylesheet file contains only styles for env component. (however please note that env is a bad class name)
However env.css contains as well definitions for .container class which is already defined in style.css